README.rst

```diff
@@ -26,6 +26,20 @@ Installation
 
     $ pip install --upgrade gcloud-{aio,rest}-{client_name}
 
+Compatibility
+-------------
+
+These libraries are used with Google projects. Here are notes on compatibility
```

Comment:
nit: `These libraries are used with Google projects.`
I don't think there is any need for this statement ~ its implied that these libraries work with GCP projects.

The blurb otherwise looks okay to me.